src/components/applicant-profile/BioCard.js

```diff
@@ -1,33 +1,23 @@
 import React from "react";
-import { useStyles } from "./ApplicantProfile.styles";
 import Paper from "@material-ui/core/Paper";
+import { useStyles } from "./ApplicantProfile.styles";
 import Grid from "@material-ui/core/Grid";
 
-const BioCard = () => {
+const BioCard = ({ applicantDetails }) => {
```

Comment:
I've wondering about the case where a user choses to submit a really short bio. Do we have some kind of min height/width or some other way of making sure they don't destroy your beautiful UI with a one sentence bio?